ui/components/DashboardComponent.js

```diff
@@ -526,17 +555,49 @@ class DashboardComponent extends React.Component {
       </div>
     );
 
+    /**
+     * getMesheryVersionText returs a well formatted version text
+     * @param {string} type type of version could be "latest" or "current"
+     */
+    const getMesheryVersionText = (type) => {
```

Comment:
Why is this function needed? All of this can be handled in showRelease itself.
We need to show `running meshery` for both cases, and for the next line use
 ```js
{outdated ? 'Latest available' : 'up-to-date'}
```

Reply:
Any way we can handle this later if you want to keep this PR simple. LGTM! 🙂